aslam_cv_calibration/include/aslam/calibration/target-checkerboard.h

```diff
@@ -0,0 +1,107 @@
+#ifndef ASLAM_CALIBRATION_TARGET_CHECKERBOARD_H
+#define ASLAM_CALIBRATION_TARGET_CHECKERBOARD_H
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <Eigen/Core>
+#include <opencv2/core/core.hpp>
+
+#include "aslam/calibration/target-base.h"
+#include "aslam/calibration/target-observation.h"
+
+namespace aslam {
+namespace calibration {
+
+/// \brief Construct a calibration target
+///        num_tag_rows:              number of tags in y-dir
+///        num_tag_cols:              number of tags in x-dir
+///        tag_size_meter:            size of a tag [m]
```

Comment:
The names above are suboptimal IMO. I would prefer the names from https://github.com/ethz-asl/kalibr/blob/14e47ea0edb7a42efa03533855e533f9e6ad7bb8/aslam_cv/aslam_cameras/src/GridCalibrationTargetCheckerboard.cpp#L12. Was there a reason to not copy these as well?

Reply:
I think I used the naming from target-aprilgrid of this repository
I'll change it to the naming of kalibr instead though